src/Repository/DescriptionRepository.php

```diff
@@ -6,14 +6,7 @@
 use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
 use Doctrine\Persistence\ManagerRegistry;
 
-/**
- * @extends ServiceEntityRepository<Description>
- *
- * @method Description|null find($id, $lockMode = null, $lockVersion = null)
- * @method Description|null findOneBy(array $criteria, array $orderBy = null)
- * @method Description[]    findAll()
- * @method Description[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
- */
+
```

Comment:
Useful DocBlocks and other comments should not be removed (cf. https://docs.phpdoc.org/guide/guides/docblocks.html).